src/software/ai/navigator/obstacle/obstacle_test.cpp

```diff
@@ -10,16 +10,16 @@
 #include <sstream>
 
 #include "shared/constants.h"
-#include "software/geom/point.h"
+#include "software/new_geom/point.h"
 
 // obstacle with robot radius factor = 1, velocity projection factor = 1
 // centred at (0,0) and with (0,0) velocity
```

Comment:
(out-of-scope, optiona): while we're here, `centred`->`centered`
``` suggestion
    // centered at (0,0) and with (0,0) velocity
```

Reply:
Oh my god, this misspelling is _everywhere_, nevermind, we can make this change all at once later...........

Reply:
Agree, perhaps a different PR for this one.

Reply:
oh interesting, TIL. this seems fine then